tests/view_tests/tests/test_debug.py

```diff
@@ -249,6 +249,15 @@ def test_technical_404_converter_raise_404(self):
             response = self.client.get('/path-post/1/')
             self.assertContains(response, 'Page not found', status_code=404)
 
+    def test_exception_reporter_from_request(self):
+        response = self.client.get('/custom_reporter_class_view/')
+        self.assertContains(response, 'custom traceback text', status_code=500)
+
+    @override_settings(DEFAULT_EXCEPTION_REPORTER='view_tests.views.CustomExceptionReporter')
+    def test_exception_reporter_from_settings(self):
+        response = self.client.get('/raises500/')
+        self.assertContains(response, 'custom traceback text', status_code=500)
```

Comment:
We can add tests for `CustomExceptionReporter.get_traceback_text()`, e.g.
```python
    def test_exception_reporter_from_settings(self):
        response = self.client.get('/raises500/', HTTP_X_REQUESTED_WITH='XMLHttpRequest')
        self.assertContains(response, 'custom traceback text', status_code=500)
```
or we can remove unused `CustomExceptionReporter.get_traceback_text()`.

Reply:
I'll remove it. It's not under test in significant way.